clippy_lints/src/utils/sugg.rs

```diff
@@ -175,15 +175,6 @@ impl<'a> Sugg<'a> {
         make_unop("&mut *", self)
     }
 
-    /// Convenience method to create the `<lhs>..<rhs>` or `<lhs>...<rhs>`
```

Comment:
Please leave that one in (maybe just keep exporting the `Sugg` type). I think we'll want to upstream it to rustc at some point and have it support a lot of expressions. Also there's an open issue about using this function on a bad suggestion currently produced due to not using this function

Reply:
https://github.com/rust-lang-nursery/rust-clippy/pull/3105/commits/d5534ca9db4f18a707ee7bc3edc1eb70647dca3b